requirements.txt

```diff
@@ -1,3 +1,3 @@
-fonduer==0.4.0
+fonduer>=0.5.0
```

Comment:
Pinning with `>=` so that this doesn't need to be updated in-between the minor version changes.